types/types.go

```diff
@@ -256,7 +256,10 @@ type Info struct {
 	Runtimes           map[string]Runtime
 	DefaultRuntime     string
 	Swarm              swarm.Info
-	LiveRestore        bool
+	// Determine whether containers should be kept running when the
```

Comment:
https://golang.org/pkg/runtime/#MemProfileRecord
https://golang.org/pkg/crypto/tls/#Config
https://golang.org/pkg/image/#Alpha16
https://godoc.org/github.com/docker/distribution#Descriptor
https://golang.org/pkg/text/scanner/#Scanner

Document the fields of a struct so we don't have to waste time figuring out what they do.